packages/core/src/lib/wallet/wallet.types.ts

```diff
@@ -113,6 +117,10 @@ export interface BrowserWalletSignAndSendTransactionsParams
 export type BrowserWalletBehaviour = Modify<
   BaseWalletBehaviour,
   {
+    importAccountsByEncryptedUrl?(
+      params: AccountImportEncryptedParams
+    ): Promise<void>;
```

Comment:
one question here, so what is expected returned promise like?
eg 1. url link. (string url, https://somewallet.io/batch-import#${hash})
eg 2. function call `window.open(url, '_blank')`

Reply:
As outcome of our discussion, once you made the change to replace `Promise<void>` to `string`, I will approve them :)